app/controllers/api_controller/initializer.rb

```diff
@@ -53,15 +57,10 @@ def log_config
       # Accessed as @attr_<type>[<name>], much faster than array include?
       #
       def gen_attr_type_hash
-        @attr_time = {}
-        @attr_url  = {}
-        @attr_resource  = {}
-        @attr_encrypted = {}
-
-        ATTR_TYPES[:time].each { |attr| @attr_time[attr] = true }
-        ATTR_TYPES[:url].each  { |attr| @attr_url[attr]  = true }
-        ATTR_TYPES[:resource].each  { |attr| @attr_resource[attr]  = true }
-        ATTR_TYPES[:encrypted].each { |attr| @attr_encrypted[attr] = true }
+        ATTR_TYPES[:time].each { |attr| normalized_attributes[:time][attr] = true }
+        ATTR_TYPES[:url].each  { |attr| normalized_attributes[:url][attr]  = true }
+        ATTR_TYPES[:resource].each  { |attr| normalized_attributes[:resource][attr]  = true }
+        ATTR_TYPES[:encrypted].each { |attr| normalized_attributes[:encrypted][attr] = true }
```

Comment:
@isimluk thanks for bringing that to my attention! Think I just found a bug - when we changed the config to a `Config::Options` it stopped responding properly to `each_value`. I'll fix that in another PR